plugins/scaffolder/api-report.md

```diff
@@ -164,6 +164,16 @@ export interface RepoUrlPickerUiOptions {
   allowedHosts?: string[];
   // (undocumented)
   allowedOwners?: string[];
+  // (undocumented)
+  requestUserCredentials?: {
+    resultSecretsKey: string;
```

Comment:
Wondering if there's a better name here? I find it to be a bit unintuitive looking at it.

`secretsKey`
`secretsOutputKey`
`saveToSecretsKey`
`secretsSaveKey`
`secretsTemplateKey`
`templateSecretsKey`

Reply:
Yeah naming this thing has been hard, `secretsKey` is probably the most I like, but to be honest I don't LOVE all of them including the one I suggested.

Let's go with `secretsKey` for now. 👍